IAM/Frequently_asked_questions.mediawiki

```diff
@@ -211,11 +211,60 @@ If you own a volunteer LDAP account, then you can upgrade your mozillians profil
 
+==== '''Q''': ''Why do the email login links expire after 15 minutes?'' ====
+
+When you login using an email link, that link is valid for 15 minutes from when you request it.
+This expiration window of 15 minutes is driven both by security considerations and a desire for
+a positive user experience. The link is short lived so that there is a limited window
+of time during which a potential attacker could use the link if they were able to get access
+to it. This is especially important due to the inherently insecure nature of email
+transmission. The user experience of having to wait for over 15 minutes to log into
+a website is one that we want to ensure our users don't experience.
+
+Experience during the past 18+ months shows that for a vast majority of users, this 15 minute expiration window has no effect on
+them as they receive the email link in their inbox mere seconds after they click the
+button requesting the link. Some users however do not receive the email login link
+immediately.
+
+Short delays in delivery are just part of how email delivery works.
+Longer delays however can be caused by a feature called [https://en.wikipedia.org/wiki/Greylisting Greylisting].
+
+==== '''Q''': ''What is email greylisting?'' ====
+Some email providers institute Greylisting on all inbound email for their users
+as a measure to reduce spam. Greylisting temporarily rejects email from mail servers
+where the sender hasn't communicated with the recipient before or due to some other
+signal indicating the email may be spam. A mail server that is Greylisting expects
+that a valid sender and mail server will continue to retry sending the email over
+time and the Greylisting mail server will eventually accept the mail. In the case
+of a "transactional" email like the email login link,
+[https://en.wikipedia.org/wiki/Greylisting#Delayed_delivery_issues Greylisting prevents users from being able to do a real-time login].
+
+ ==== '''Q''': ''What can I do about my email provider's greylisting?'' ==== 
+Users who have email providers that use Greylisting will likely see this type of
+severely delayed transactional emails from other senders as well, for example
+when they sign up to a new web site and that web site sends them an email with a
+link to confirm that the user controls the email address they signed up with.
+
+Unfortunately, in order for users with email providers that utilize Greylisting
+to work around this problem, they may need to contact their email provider or
+look in their providers documentation to see if there are whitelisting options
+available to them. There's nothing that Mozilla can do on the sending side to
+force the user's Greylisting mail server to accept and deliver the email.
+
+Users sometime ask to just increase the expiration time, for example to 30 or 60
+minutes. We deliberately have not made this change because doing so would decrease our systems' security while not addressing the root cause of this problem.
```

Comment:
```suggestion
minutes. We deliberately have not made this change because doing so would decrease our systems' security while not addressing the root cause of this problem (Greylisting).
```